sdk/core/core-https/README.md

```diff
@@ -0,0 +1,69 @@
+# Azure Core HTTP client library for JavaScript
+
+This is the core HTTP pipeline for Azure SDK JavaScript libraries which work in the browser and Node.js. This library is primarily intended to be used in code generated by [AutoRest](https://github.com/Azure/Autorest) and [`autorest.typescript`](https://github.com/Azure/autorest.typescript).
+
+## Getting started
+
+### Requirements
+
+- [Node.js](https://nodejs.org) version > 8.x
+
+### Installation
+
+This package is primarily used in generated code and not meant to be consumed directly by end users.
+
+## Key concepts
+
+You can find an explanation of how this repository's code works by going to our [architecture overview](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/core/core-http/docs/architectureOverview.md).
```

Comment:
are we going to have a new architecture overview instead of linking to the old one?

Reply:
Yeah, I'll put a TODO